pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -469,7 +469,7 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1beta1.PipelineRun) err
 		return err
 	}
 
-	after := resources.GetPipelineConditionStatus(pr, pipelineState, logger, d, dfinally)
+	after := pipelineState.GetPipelineConditionStatus(pr, logger, d, dfinally)
```

Comment:
this is referred to as `pipelinestate` sometimes and `pipelinerunstate` at other times -- i think we should update to have a consistent name

Reply:
thanks @jerop I have renamed `pipelinestate` to `pipelineRunState` which is of type `PipelineRunState` in `pipelinerun.go`. Also, Have left the receiver as `state` in `pipelinerunstate.go` for simplicity.